<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <widget class="QPushButton" name="btnSend">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	color:#000000;

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 185, 240, 255), stop:0.4 rgba(0, 150, 250, 255), stop:1 rgba(255, 255, 255, 255));
	border:1px solid rgb(115, 205, 240);
	border-top-left-radius: 30px; /* 左上角设置为圆角 */
   border-top-right-radius: 12px; /* 右上角设置为圆角 */
   border-bottom-left-radius: 12px; /* 左下角设置为圆角 */
   border-bottom-right-radius: 30px; /* 右下角设置为圆角 */
    
}</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnReceive">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.119403 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));
    border:1px solid rgb(255, 255, 255);
    border-top-left-radius: 30px; /* 左上角设置为圆角 */
    border-top-right-radius: 12px; /* 右上角设置为圆角 */
    border-bottom-left-radius: 12px; /* 左下角设置为圆角 */
    border-bottom-right-radius: 30px; /* 右下角设置为圆角 */
}
</string>
        </property>
        <property name="text">
         <string>接受</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>8</y>
          <width>361</width>
          <height>501</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QStackedWidget
{
	
	background-color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QPushButton" name="btnSendFolder">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>300</y>
            <width>90</width>
            <height>30</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	color:#000000;

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 185, 240, 255), stop:0.4 rgba(0, 150, 250, 255), stop:1 rgba(255, 255, 255, 255));
	border:1px solid rgb(115, 205, 240);
	border-radius: 10px;
	
    
}
QPushButton:hover
{
	color:#ffffff;
	background-color:rgb(115, 205, 240);
	border-radius: 10px;
	padding: 6px 12px;
}
QPushButton:pressed
{
	color:#55aaff;
	background-color:rgb(255, 255, 255);
	border-radius: 10px;
	padding: 6px 12px;
}</string>
          </property>
          <property name="text">
           <string>Select Folder</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>50</y>
            <width>100</width>
            <height>90</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../res.qrc">:/icons/res/send.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSendFile">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>300</y>
            <width>90</width>
            <height>30</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	color:#000000;

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(40, 185, 240, 255), stop:0.4 rgba(0, 150, 250, 255), stop:1 rgba(255, 255, 255, 255));
	border:1px solid rgb(115, 205, 240);
	border-radius: 10px;
	
    
}
QPushButton:hover
{
	color:#ffffff;
	background-color:rgb(115, 205, 240);
	border-radius: 10px;
	padding: 6px 12px;
}
QPushButton:pressed
{
	color:#55aaff;
	background-color:rgb(255, 255, 255);
	border-radius: 10px;
	padding: 6px 12px;
}</string>
          </property>
          <property name="text">
           <string>Select File</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>160</y>
            <width>160</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Impact</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit 
{
	border:none;
}</string>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapColumnOrWidth">
           <number>0</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Impact'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-size:18pt; font-weight:700; color:#55aaff;&quot;&gt;发送文件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>240</y>
            <width>280</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QTextEdit" name="textEdit_2">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>160</y>
            <width>160</width>
            <height>40</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Impact</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit 
{
	border:none;
}</string>
          </property>
          <property name="undoRedoEnabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Impact'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-size:18pt; font-weight:700; color:#ffaa00;&quot;&gt;接收文件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>50</y>
            <width>100</width>
            <height>90</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../res.qrc">:/icons/res/download.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>200</y>
            <width>180</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
	border:none;
}</string>
          </property>
          <property name="text">
           <string>请在下方输入正确的接收密码</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="editCode">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>240</y>
            <width>280</width>
            <height>25</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit ::placeholder 
{
	color:gray;
}
</string>
          </property>
          <property name="text">
           <string>code</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="btnReceiveFile">
          <property name="geometry">
           <rect>
            <x>125</x>
            <y>300</y>
            <width>90</width>
            <height>30</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	color:#000000;


	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.119403 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));
	border:1px solid rgb(255,170,0);
	border-radius: 10px;
	
    
}
QPushButton:hover
{
	color:#ffffff;
	background-color:rgb(255,170,0);
	border-radius: 10px;
	padding: 6px 12px;
}
QPushButton:pressed
{
	color:#ffaa00;
	background-color:rgb(255, 255, 255);
	border-radius: 10px;
	padding: 6px 12px;
}</string>
          </property>
          <property name="text">
           <string>Receive File</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
